Program.cs

```diff
@@ -1,10 +1,36 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// Convert decimal to hexadecimal and binary
+//string hexvalue = input.ToString ("X") ----> in-built function for reference
+
+Console.Write ("Enter a decimal value: ");
+if (int.TryParse (Console.ReadLine (), out int input)) {
+   if (input == 0) Console.WriteLine ("Hexadecimal: 0 \nBinary: 0");

```

Comment:
No need a if statement you handle the check inside the writeline for both the conditions using ternary operator.